From b213db6e0ef02fad3f0f9813c01729fd86e3c79c Mon Sep 17 00:00:00 2001
From: Valentin Raevsky <valentin@compulab.co.il>
Date: Tue, 26 Feb 2019 14:42:02 +0200
Subject: [PATCH 5/5] cl-som-imx8: dtb: Update Makefile

Signed-off-by: Valentin Raevsky <valentin@compulab.co.il>
---
 arch/arm64/boot/dts/compulab/Makefile | 2 --
 1 file changed, 2 deletions(-)

diff --git a/arch/arm64/boot/dts/compulab/Makefile b/arch/arm64/boot/dts/compulab/Makefile
index b9cc045..4953903 100644
--- a/arch/arm64/boot/dts/compulab/Makefile
+++ b/arch/arm64/boot/dts/compulab/Makefile
@@ -1,6 +1,4 @@
 dtb-$(CONFIG_ARCH_FSL_IMX8MQ) += cl-som-imx8.dtb
-dtb-$(CONFIG_ARCH_FSL_IMX8MQ) += cl-som-imx8-uarts.dtb
-dtb-$(CONFIG_ARCH_FSL_IMX8MQ) += cl-som-imx8-ecspi.dtb
 dtb-$(CONFIG_ARCH_FSL_IMX8MQ) += sbc-imx8.dtb
 dtb-$(CONFIG_ARCH_FSL_IMX8MQ) += sbc-imx8-thermal.dtb
 dtb-$(CONFIG_ARCH_FSL_IMX8MQ) += sbc-imx8-bluetooth.dtb
-- 
1.9.1

