From b8a56120f589d3d642fa7ed75eac35231df1c058 Mon Sep 17 00:00:00 2001
From: Valentin Raevsky <valentin@compulab.co.il>
Date: Mon, 24 Jun 2019 08:01:01 +0300
Subject: [PATCH 7/8] cl-som-imx8: pmic: Increase DRAM power rail voltage

Signed-off-by: Valentin Raevsky <valentin@compulab.co.il>
---
 board/compulab/cl-som-imx8/spl.c | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/board/compulab/cl-som-imx8/spl.c b/board/compulab/cl-som-imx8/spl.c
index 4b4a2d0..4513fb7 100644
--- a/board/compulab/cl-som-imx8/spl.c
+++ b/board/compulab/cl-som-imx8/spl.c
@@ -186,6 +186,14 @@ int power_init_board(void)
 	pmic_reg_read(p, PFUZE100_DEVICEID, &reg);
 	printf("PMIC:  PFUZE100 ID=0x%02x\n", reg);
 
+	/* Increase the DRAM rail voltage up to 1V125 */
+	pmic_reg_read(p, PFUZE100_SW2VOL, &reg);
+	if ((reg & 0x3f) != 0x1E) {
+		reg &= ~0x3f;
+		reg |= 0x1E;
+		pmic_reg_write(p, PFUZE100_SW2VOL, reg);
+	}
+
 	/* Increase the GPU rail voltage */
 	pmic_reg_read(p, PFUZE100_SW1ABVOL, &reg);
 	if ((reg & 0x3f) != 0x1C) {
-- 
1.9.1

