From 372cba5e805eda8c4cac462c29f13ea2b893947e Mon Sep 17 00:00:00 2001
From: Ilya Ledvich <ilya@compulab.co.il>
Date: Sun, 11 Feb 2018 14:39:37 +0200
Subject: [PATCH 38/51] ARM: imx7d: clk: select uart7 clock parent and set rate

Signed-off-by: Ilya Ledvich <ilya@compulab.co.il>
---
 drivers/clk/imx/clk-imx7d.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/clk/imx/clk-imx7d.c b/drivers/clk/imx/clk-imx7d.c
index f5a33446..c506da9 100644
--- a/drivers/clk/imx/clk-imx7d.c
+++ b/drivers/clk/imx/clk-imx7d.c
@@ -930,6 +930,8 @@ static void __init imx7d_clocks_init(struct device_node *ccm_node)
 	imx_clk_set_rate(clks[IMX7D_UART5_ROOT_DIV], 80000000);
 	imx_clk_set_parent(clks[IMX7D_UART6_ROOT_SRC], clks[IMX7D_PLL_SYS_MAIN_240M_CLK]);
 	imx_clk_set_rate(clks[IMX7D_UART6_ROOT_DIV], 80000000);
+	imx_clk_set_parent(clks[IMX7D_UART7_ROOT_SRC], clks[IMX7D_PLL_SYS_MAIN_240M_CLK]);
+	imx_clk_set_rate(clks[IMX7D_UART7_ROOT_DIV], 80000000);
 
 	imx_register_uart_clocks(uart_clks);
 }
-- 
1.9.1

