From de0f3653268c4a49738ebe1cab57429f23fda2a2 Mon Sep 17 00:00:00 2001
From: Valentin Raevsky <valentin@compulab.co.il>
Date: Thu, 11 Oct 2018 14:01:50 +0300
Subject: [PATCH 81/86] imx8: Makefile: Update dts list

Signed-off-by: Valentin Raevsky <valentin@compulab.co.il>
---
 arch/arm64/boot/dts/compulab/Makefile | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/arch/arm64/boot/dts/compulab/Makefile b/arch/arm64/boot/dts/compulab/Makefile
index 40e2b94..51ca4fb 100644
--- a/arch/arm64/boot/dts/compulab/Makefile
+++ b/arch/arm64/boot/dts/compulab/Makefile
@@ -1,7 +1,4 @@
 dtb-$(CONFIG_ARCH_FSL_IMX8MQ) += cl-som-imx8.dtb
-dtb-$(CONFIG_ARCH_FSL_IMX8MQ) += cl-som-imx8-lvds-dcss.dtb
-dtb-$(CONFIG_ARCH_FSL_IMX8MQ) += cl-som-imx8-lvds-lcdif.dtb
-dtb-$(CONFIG_ARCH_FSL_IMX8MQ) += cl-som-imx8-dual.dtb
 dtb-$(CONFIG_ARCH_FSL_IMX8MQ) += sbc-imx8.dtb
 dtb-$(CONFIG_ARCH_FSL_IMX8MQ) += sbc-imx8-lvds-dcss.dtb
 dtb-$(CONFIG_ARCH_FSL_IMX8MQ) += sbc-imx8-lvds-lcdif.dtb
-- 
1.9.1

