From 4e8367a50d6c1fd15598cdf38f5cd8be6dfc76b8 Mon Sep 17 00:00:00 2001
From: Valentin Raevsky <valentin@compulab.co.il>
Date: Tue, 30 Jan 2018 12:40:31 +0200
Subject: [PATCH 42/64] arm64: Enable SPI_IMX for ARM64

Signed-off-by: Valentin Raevsky <valentin@compulab.co.il>
---
 drivers/spi/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/spi/Kconfig b/drivers/spi/Kconfig
index c285948..c51dd7a 100644
--- a/drivers/spi/Kconfig
+++ b/drivers/spi/Kconfig
@@ -290,7 +290,7 @@ config SPI_IMG_SPFI
 
 config SPI_IMX
 	tristate "Freescale i.MX SPI controllers"
-	depends on ARCH_MXC || COMPILE_TEST
+	depends on ARCH_MXC || COMPILE_TEST || ARCH_MXC_ARM64
 	select SPI_BITBANG
 	help
 	  This enables using the Freescale i.MX SPI controllers in master
-- 
1.9.1

